src/insns/prefetch.i.adoc

```diff
@@ -49,6 +44,13 @@ likely to be accessed by an instruction fetch in the near future. The encoding
 is only valid if imm[4:0]=0. The authorising capability for this operation is
 <<ddc>>.
 
+In either mode, instruction does not perform a memory access
```

Comment:
Looks like there is a 'the' missing before 'instruction', or even better if you replace 'instruction' with 'prefetch.i'